### PR TITLE
Feature/full edit event form

### PR DESCRIPTION
**Pull request by chooidos**

### CODE REVIEW

File: client/src/shared/utilities/validation.js [hunk 1]
```diff
@@ -26,9 +26,11 @@ const validatorsDictionary = {
...
-  [REQ_NUMBERS]: value => {
+  [REQ_NUMBERS]: (value, validator) => {
     const expression = /^\d*[1-9]\d*$/;
-    return expression.test(value);
+    const minValidation = validator.min ? value >= validator.min : true;
+    const maxValidation = validator.max ? value <= validator.max : true;
+    return expression.test(value) && minValidation && maxValidation;
   },
   [REQ_EMAIL]: value => {
     // eslint-disable-next-line
```
Comment (yushevchuk) [inline client/src/shared/utilities/validation.js]: Remove comment, please.

File: server/middlewares/validator.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ const { body, validationResult } = require('express-validator');
 const regExpForPassword = /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])/;
 //Cyrillic and Latin characters, space and -
```
Comment (yushevchuk) [inline server/middlewares/validator.js]: Remove comment, please.

File: client/src/shared/utilities/validation.js (hunk 1 above)
Comment (chooidos) [reply client/src/shared/utilities/validation.js]: this comment is really needed here

File: server/middlewares/validator.js (hunk 2 above)
Comment (chooidos) [reply server/middlewares/validator.js]: this is not mine) but ok

**Comment by chooidos** [comment]
> Old cover (file) will not be deleted on backend

done

File: server/middlewares/validator.js (hunk 2 above)
Comment (chooidos) [reply server/middlewares/validator.js]: done

**Pull request merged**